com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -484,7 +489,7 @@ internal void InitializeVariables()
         private static ProfilerMarker s_NetworkBehaviourUpdate = new ProfilerMarker($"{nameof(NetworkBehaviour)}.{nameof(NetworkBehaviourUpdate)}");
 #endif
 
-        internal static void NetworkBehaviourUpdate(NetworkManager networkManager)
+        internal static void NetworkBehaviourUpdate(NetworkManager networkManager, SnapshotSystem snapshot)
```

Comment:
Consider making SnapshotManager a member of NetworkManager?

Reply:
It already is, yes, I see your point. One less parameter.